src/statics.ts

```diff
@@ -62,6 +62,10 @@ export abstract class Statics {
   static ssmFormOverviewTableArn: string = `/${this.projectName}/formOverviewTableArn`;
   static ssmFormOverviewTableName: string = `/${this.projectName}/formOverviewTableName`;
   static ssmFormIoBaseUrl: string = `/${this.projectName}/formIoBaseUrl`;
+  static ssmJwtTokenIssuer = `/${this.projectName}/jwtTokenIssuer`;
+  static ssmYiviClaimBsn = `/${this.projectName}/yiviClaimBsn`;
```

Comment:
Deze en de 2 regels hieronder kunnen weg volgens mij. We hebben de logica mbt het signicat token hier niet meer nodig